qube/create/domains/LibCreator.py

```diff
@@ -26,16 +26,16 @@ def __init__(self):
         self.TEMPLATES_LIB_PATH = f'{self.WD_Path.parent}/templates/lib'
 
         '"" TEMPLATE VERSIONS ""'
-        self.LIB_JAVA_TEMPLATE_VERSION = load_qube_template_version('lib-java', self.AVAILABLE_TEMPLATES_PATH)
+        self.LIB_JAVA_TEMPLATE_VERSION = load_qube_template_version('lib-groovy', self.AVAILABLE_TEMPLATES_PATH)
```

Comment:
No!
Keep the java here, please
Add a new line:

`self.LIB_GROOVY_TEMPLATE_VERSION = load_qube_template_version('lib-groovy', self.AVAILABLE_TEMPLATES_PATH)`

Reply:
You have to adapt

```
 # switch case statement to fetch the template version
        switcher_version = {
            'java': self.LIB_JAVA_TEMPLATE_VERSION
        }
```
And add groovy here
You can imagine what it has to look like :)

Reply:
Fixed 2312dd6
thanks for you hint!